aspnet/signalr/overview/getting-started/tutorial-high-frequency-realtime-with-signalr.md

```diff
@@ -225,7 +225,7 @@ This code moves the shape from the old location to the new one. The server gives
 
 ## Additional resources
 
-The communication paradigm you just learned about is useful for developing online games and other simulations, like [the ShootR game created with SignalR](https://shootr.azurewebsites.net/).
+The communication paradigm you just learned about is useful for developing online games and other simulations, like the ShootR game created with SignalR.
```

Comment:
```suggestion
```

Reply:
Since there is no ShootR game example to point to and it is not part of this tutorial, I think it would be best to not reference it at all.